### ISSUE TITLE
Any plan for dataset release?

### ISSUE THREAD

**Issue by yinkangning0124** [IDEA-Research/MotionLLM#3]
Hi Linghao,
    What an excellent work! I wonder is there any plan to release the dataset in this paper? 
Thanks a lot,
Kangning.

**Comment by LinghaoChan** [comment]
> Hi Linghao, What an excellent work! I wonder is there any plan to release the dataset in this paper? Thanks a lot, Kangning.

Yes. We are working on this.

**Comment by yinkangning0124** [comment]
> > Hi Linghao, What an excellent work! I wonder is there any plan to release the dataset in this paper? Thanks a lot, Kangning.
> 
> Yes. We are working on this.

Looking forward to this ! Thanks again for your great effort.

**Comment by LinghaoChan** [comment]
> > > Hi Linghao, What an excellent work! I wonder is there any plan to release the dataset in this paper? Thanks a lot, Kangning.
> > 
> > 
> > Yes. We are working on this.
> 
> Looking forward to this ! Thanks again for your great effort.

@yinkangning0124 Do you have any suggestions on the schedule (like ''before xxx''). Or which part of the dataset will benefit you most? Because of our schedule on releasing training codes, evaluation, benchmark, and datasets are a bit conflict, your suggestion might be helpful to arrange our time.

**Comment by yinkangning0124** [comment]
> > > > Hi Linghao, What an excellent work! I wonder is there any plan to release the dataset in this paper? Thanks a lot, Kangning.
> > > 
> > > 
> > > Yes. We are working on this.
> > 
> > 
> > Looking forward to this ! Thanks again for your great effort.
> 
> @yinkangning0124 Do you have any suggestions on the schedule (like ''before xxx''). Or which part of the dataset will benefit you most? Because of our schedule on releasing training codes, evaluation, benchmark, and datasets are a bit conflict, your suggestion might be helpful to arrange our time.

Yes, I'm quite interested in the annotated Motion-X dataset, which I think is the motion-video-text pair right? May be you can release the corresponding dataset before 20th June if possible.
Thanks again for your great effort !

**Comment by LinghaoChan** [comment]
> > > > > Hi Linghao, What an excellent work! I wonder is there any plan to release the dataset in this paper? Thanks a lot, Kangning.
> > > > 
> > > > 
> > > > Yes. We are working on this.
> > > 
> > > 
> > > Looking forward to this ! Thanks again for your great effort.
> > 
> > 
> > @yinkangning0124 Do you have any suggestions on the schedule (like ''before xxx''). Or which part of the dataset will benefit you most? Because of our schedule on releasing training codes, evaluation, benchmark, and datasets are a bit conflict, your suggestion might be helpful to arrange our time.
> 
> Yes, I'm quite interested in the annotated Motion-X dataset, which I think is the motion-video-text pair right? May be you can release the corresponding dataset before 20th June if possible. Thanks again for your great effort !

ok, ic! Thanks for your suggestion.

**Comment by LinghaoChan** [comment]
@yinkangning0124 Could you please provide more details about your demands? Because some of the data has some copyright issues, we cannot directly redistribute the data. We are considering this. If you can provide your detailed request, I will act accordingly. If it is not very convenient to detail it in public, please reach out to me via my public email.

**Comment by AQFU** [comment]
Hoping "the motion demo of MotionLLM"

**Comment by LinghaoChan** [comment]
> > > > > Hi Linghao, What an excellent work! I wonder is there any plan to release the dataset in this paper? Thanks a lot, Kangning.
> > > > 
> > > > 
> > > > Yes. We are working on this.
> > > 
> > > 
> > > Looking forward to this ! Thanks again for your great effort.
> > 
> > 
> > @yinkangning0124 Do you have any suggestions on the schedule (like ''before xxx''). Or which part of the dataset will benefit you most? Because of our schedule on releasing training codes, evaluation, benchmark, and datasets are a bit conflict, your suggestion might be helpful to arrange our time.
> 
> Could you public the H3DQA please?

On the way. Thx. There are a bag of work for us to do. The priority of H3DQA might be a month later.

**Comment by LinghaoChan** [comment]
@yinkangning0124 We plan to release the MoVid video QA part in the coming week.

**Comment by LinghaoChan** [comment]
We plan to announce the video part data release next Monday officially. If any other further situations, we may also release data before that time.

**Comment by LinghaoChan** [comment]
See dataset preview [here](https://huggingface.co/spaces/EvanTHU/Movid-vis).

**Comment by Chatonz** [comment]
> https://github…
I tried to access the dataset but the webpage returned a 404 error.

**Comment by Chatonz** [comment]
> [此处](https://huggingface.co/spaces/EvanTHU/Movid-vis)查看数据集预览。

The dataset looks really great. Do you have any plans to release all of it?

**Comment by LinghaoChan** [comment]
> > [此处](https://huggingface.co/spaces/EvanTHU/Movid-vis)查看数据集预览。
> 
> The dataset looks really great. Do you have any plans to release all of it?

The dataset is already prepared for release. We are resolving some legal reviewing issues. Perhaps two weeks.

**Comment by AshkanTaghipour** [comment]
Hi, thank you for the novel task.
is there any plan for releasing annotated Motion-X dataset?

**Comment by LinghaoChan** [comment]
@AshkanTaghipour Please refer to the readme. The link is https://huggingface.co/datasets/EvanTHU/MoVid

**Comment by LinghaoChan** [comment]
> Thank you for your prompt reply. Upon reviewing the dataset, I noticed it includes only videos without any annotated motion, correct?

The motion comes from Motion-X. You can download them accordingly.

**Comment by qhFang** [comment]
Thank you for your efforts. However, I noticed that the amount of data currently released differs significantly from what was claimed in the paper. The paper claims that 200k QA pairs were generated based on the Motion-X dataset, but there are only 24K QA pairs in video-QA.json. Are there any plans to release the remaining data in the future?

**Comment by LinghaoChan** [comment]
> Thank you for your efforts. However, I noticed that the amount of data currently released differs significantly from what was claimed in the paper. The paper claims that 200k QA pairs were generated based on the Motion-X dataset, but there are only 24K QA pairs in video-QA.json. Are there any plans to release the remaining data in the future?

@qhFang The released dataset part is the captioning part. We plan to release the QA data in next month.

**Comment by palmex** [comment]
> priority of H3DQA might be a month later.

Were you able to release the HumanML3D Q&A dataset? I'm not looking for the videos, but just the Q&A. 

Also, did you have GPT-4V label the SMPL renderings of this dataset? I don't believe the original videos of HumanML3D are available.
